MERGETOOL.md

```diff
@@ -8,8 +8,8 @@ The first step is to configure git so that it knows how to invoke your merge too
 
 ```ini
 [mergetool "extMerge"]
-	cmd = extMergeTool "$BASE" "$LOCAL" "$REMOTE" "$MERGED"
-	trustExitCode = false
+  cmd = extMergeTool "$BASE" "$LOCAL" "$REMOTE" "$MERGED"
+  trustExitCode = false
```

Comment:
Not sure how much this matters but in git config tabs are used instead of spaces: https://git-scm.com/docs/git-config

Reply:
Reverted the git config lines.